emperor/support_files/js/emperor.js

```diff
@@ -886,20 +888,22 @@ function toggleTaxaLabels(){
 	}
 }
 
-/* Turn on and off the spheres representing the biplots on screen */
-function toggleBiplotVisibility(){
+/* Turn on and off the spheres representing the biplots on screen
+If the toggleArrow variable is true, then only the visibility of the taxa vectors will be altered
+Otherwise, only the visibility of the */
```

Comment:
Can you remove this line?

While we don't have any guidelines on this, we try to keep the comments formatted in the following way:

```javascript
/* Turn on and off the spheres representing the biplots on screen

   If the toggleArrow variable is true, then only the visibility of the taxa vectors will be altered.
*/
```